pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -224,7 +287,7 @@ func extractLibInfo(pod *corev1.Pod, containerRegistry string) ([]libInfo, bool)
 	var autoDetected = false
 
 	// Inject all libraries if Single Step Instrumentation is enabled
-	if isApmInstrumentationEnabled(pod.Namespace) {
+	if isApmInstrumentationEnabled() {
```

Comment:
We need the webhook because the code in `auto_instrumentation.go` also injects libraries in some cases when `isApmInstrumentationEnabled()` returns false. See this for example:
https://github.com/DataDog/datadog-agent/blob/6131465eaf383107e6adf5a5917800d6291f45d0/pkg/clusteragent/admission/mutate/auto_instrumentation.go#L163